WinUIGallery/ControlPages/IconElementPage.xaml

```diff
@@ -32,7 +32,7 @@
                 <StackPanel>
                     <TextBlock Text="Use FontIcon as the icon for a control if you want to specify a Glyph value from a FontFamily. Windows 10 uses the Segoe MDL2 Assets FontFamily and that is what this example is showing." Style="{ThemeResource BodyTextBlockStyle}" Margin="0,0,0,12"  />
```

Comment:
I think we need to update this info here. Maybe just remove it? @niels9001 @karenbtlai thoughts?

Reply:
Maybe update this with the explaination about Segoe MDL2 Assets being the default on W10 and Segoe Fluent Icons on W11?

Reply:
Sounds good to me. @jay-o-way do you want to include this in your PR?

Reply:
Hm, actually, the header **explicitly** says "...a glyph from a specific font family..." so in that context I should be reverting this specific example.